core/iwasm/common/gc/gc_common.c

```diff
@@ -14,15 +15,39 @@
 uint32
 wasm_get_defined_type_count(WASMModuleCommon *const module)
 {
-    /* TODO */
-    return 0;
+    WASMModule *wasm_module;
+    uint32 type_count = 0;
+
+#if WASM_ENABLE_INTERP != 0
+    if (module->module_type == Wasm_Module_Bytecode) {
+        wasm_module = (WASMModule *)module;
+        type_count = wasm_module->type_count;
+    }
+#endif
+
+#if WASM_ENABLE_AOT != 0
+    // TODO
+#endif
+    return type_count;
```

Comment:
Had better add an empty line before the return statement

```suggestion

    return type_count;
```

Reply:
> Had better add an empty line before the return statement

get it, thanks